src/main.py

```diff
@@ -86,10 +90,11 @@ def get_overrided_backend_config(original_config: Union[DictConfig, BackendConfi
 def run(config: BenchmarkConfig) -> None:
     backend_name = config.backend._target_.split('.')[-1].replace('Backend', '')
     if backend_name == 'PyTorch':
-        from backends.pytorch import PyTorchConfig
-        OmegaConf.register_new_resolver("pytorch_version", PyTorchConfig.version)
-        cs.store(group="backend", name="pytorch_backend", node=PyTorchConfig)
-        cs.store(group="backend", name="torchscript_backend", node=PyTorchConfig)
+        print()
+        #from backends.pytorch import PyTorchConfig
+        #OmegaConf.register_new_resolver("pytorch_version", PyTorchConfig.version)
+        #cs.store(group="backend", name="pytorch_backend", node=PyTorchConfig)
+        #cs.store(group="backend", name="torchscript_backend", node=PyTorchConfig)
```

Comment:
Can you explain why we are commenting this out? It seems to me we need to have all the registered configurations in order to support all the backends with Hydra support to keep track of everything